schemas/x509.yml

```
@@ -0,0 +1,193 @@
+---
+- name: x509
+  title: x509 Certificate
+  group: 2
+  short_description: These fields contain x509 certificate metadata.
+  description: >
+    This implements the common core fields for x509 certificates. This information is likely logged with TLS sessions,
```

```suggestion
    These fields contain x509 certificate metadata. This information is likely logged with TLS sessions,
```

The current first sentence reads like a PR body ("we're implementing this"). Let's just reuse the `short` definition as the first sentence instead. The `short` and `description` are never displayed in the same place, so it's fine to reuse it as is.